host/src/FunctionsNetHost/Grpc/WorkerConfig/WorkerDescription.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace FunctionsNetHost.Grpc
+{
+    public sealed class WorkerDescription
+    {
+        public string? DefaultWorkerPath { set; get; }
+
+        public bool IsSpecializable { set; get; }
```

Comment:
Will be updating the worker.sdk package to include this new property in the worker.config.